WordPress/Classes/ViewRelated/Pages/PageListViewController.swift

```diff
@@ -59,6 +63,9 @@ class PageListViewController : AbstractPostListViewController, UIViewControllerR
     // MARK: - UIViewController
 
     override func prepareForSegue(segue: UIStoryboardSegue, sender: AnyObject?) {
+        super.refreshNoResultsView = { [weak self] noResultsView in
```

Comment:
Okay, this is new to me. `variables` in... also allows you to grab instance variabes? are those retained?